design/src/rfcs/rfc0016_body_callback_apis_v2.md

```diff
@@ -58,6 +58,9 @@ const CRC_32_C_HEADER_NAME: &str = "x-amz-checksum-crc32c";
 const SHA_1_HEADER_NAME: &str = "x-amz-checksum-sha1";
 const SHA_256_HEADER_NAME: &str = "x-amz-checksum-sha256";
 
+const WITH_OPTIONAL_WHITESPACE: bool = true;
+const TRAILER_SEPARATOR: &str = if WITH_OPTIONAL_WHITESPACE { ": " } else { ":" };
```

Comment:
Is this configurable in the conversion of `http::Request` into the actual HTTP request headers?

Reply:
It's not configurable and I don't think it will need to be. I'm expecting that Hyper decides whether or not to include the OWS and that it never changes but I have to go check. I put this code in there so that I don't forget to verify this.